src/components/CartItem.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import Image from "next/image";
+import { BsTrash3 } from "react-icons/bs";
+import cartData from "./cartData"; // Import cartData
+
+const CartItem = ({ item, onDecrease, onIncrease, onDelete }) => {
+  const decreaseQuantity = () => {
+    onDecrease();
+  };
+
+  const increaseQuantity = () => {
+    onIncrease();
+  };
+
+  const handleDelete = () => {
+    // Call onDelete to remove the item from the cartData
+    window.location.reload();
```

Comment:
why are we using reload? nextjs or reactjs should refresh the component on every state management